packages/cms-cli/lib/createModulePrompt.js

```diff
@@ -0,0 +1,34 @@
+const inquirer = require('inquirer');
+
+const MODULE_LABEL_PROMPT = {
+  name: 'moduleLabel',
+  message: 'What should the module label be?',
+  validate(val) {
+    if (typeof val !== 'string') {
+      return 'You entered an invalid name. Please try again.';
+    } else if (!val.length) {
+      return 'The name may not be blank. Please try again.';
+    }
+    return true;
+  },
+};
+const CONTENT_TYPES_PROMPT = {
+  type: 'checkbox',
+  name: 'contentTypes',
+  message: 'What types of content will this module be used in?',
+  default: 'PAGE',
+  choices: [
+    { name: 'Page', value: 'PAGE' },
+    { name: 'Blog post', value: 'BLOG_POST' },
+    { name: 'Blog listing', value: 'BLOG_LISTING' },
+    { name: 'Email', value: 'EMAIL' },
```

Comment:
I wonder if we should automatically leave out `module.css` and `module.js` if a user checks `EMAIL` since `module.css` and `module.js` are not allowed in email.

Reply:
👍 that's a good call